htlcswitch/link.go

```diff
@@ -384,6 +384,18 @@ type channelLink struct {
 	// respect to the quiescence protocol.
 	quiescer
 
+	// activeQuiescenceRequest is a possibly nil channel that we should
```

Comment:
FWIW, sending on a `nil` channels is a panic.

Reply:
We check the channel for nil-ness prior to sending on it.